src/polls/models/pollDTO.model.ts

```diff
@@ -1,6 +1,6 @@
-import { PollStatusType, PollsResults } from '@/store/polls/types';
+import { PollStatusType, PollsResults } from '@/polls/store/types';
 import { HouseDTOModel } from '@/houses/models/houseDTO.model';
-import { PollModel, PollModelWithResults } from '@/store/polls/models/poll.model';
+import { PollModel, PollModelWithResults } from './poll.model';
```

Comment:
Didn't it should be imported from:
`'@/polls/model/poll.model'`

instead?

Reply:
It's better to put it to the '@/poll/store/model/poll.model'

Reply:
It looks better (unficied) with '@/polls/models/poll.model'. Thank you, corrected!